dune/actions/battle/resolve.py

```diff
@@ -199,7 +200,7 @@ def _execute(self, game_state):
                 ops.tank_unit(new_game_state, loser, space, sec, u)
 
         new_game_state.round_state.stage_state.winner = winner
-        new_game_state.pause.extend(battle_id[:2])
+        new_game_state.pause.append(loser)
```

Comment:
I'll have to check, but I think if the winner dialed 0 and didn't play any cards, they might need a pause

Reply:
Ah, currently they get a blank discard.  I think pausing is more appropriate though, I'll make it work like that.